client/src/lib/utils/networkData.ts

```diff
@@ -0,0 +1,27 @@
+export interface ChainData {
+	name: string;
+	id: number;
+}
+
+interface NetworkData {
+	networkName: string;
+	chains: ChainData[];
+	getChainName(id: number): string | null;
+}
+
+export const Rococo: NetworkData = {
+	networkName: "Rococo",
+	chains: [
+		{ name: "Relay Chain", id: -1 },
+		{ name: "One", id: 1 },
+		{ name: "Two", id: 2 },
+		{ name: "Three", id: 3000 }
```

Comment:
@paritytech/opstooling which chains should we add here?

Reply:
I don't know how to get a list of all possible ones, but perhaps scrolling through the faucet matrix chats and seeing which parachains people use would be a good trail?